PowerPointLabs/PowerPointLabs/PositionsLab/PositionsPaneWPF.xaml.cs

```diff
@@ -1733,7 +1733,7 @@ private ShapeRange DuplicateShapes(ShapeRange range)
             {
                 Shape shape = range[i + 1];
                 Shape duplicated = shape.Duplicate()[1];
-                duplicated.Name = shape.Name + "_Copy";
+                duplicated.Name = shape.Name + "_Copy_" + i.ToString();
```

Comment:
Possible to also leave a comment here stating the need to add the index to the name for future reference